specification/azurestack/resource-manager/Microsoft.AzureStack/preview/2020-06-01-preview/EdgeSubscription.json

```diff
@@ -0,0 +1,403 @@
+            },
+            {
+              "$ref": "#/parameters/ApiVersionParameter"
+            },
+            {
+              "$ref": "#/parameters/NewEdgeSubscriptionParameter"
+            }
+          ],
+          "responses": {
+            "200": {
+              "description": "OK",
+              "schema": {
+                "$ref": "#/definitions/EdgeSubscription"
+              }
+            },
+            "201": {
+              "description": "CREATED",
+              "schema": {
+                "$ref": "#/definitions/EdgeSubscription"
+              }
+            },
+            "default": {
+              "description": "Error response describing why the operation failed.",
+              "schema": {
+                "$ref": "#/definitions/ErrorResponse"
+              }
+            }
+          }
+        }
+      }
+    },
+    "definitions": {
+      "TrackedResource": {
+        "description": "Base resource object.",
+        "type": "object",
+        "properties": {
+          "id": {
+            "readOnly": true,
+            "description": "ID of the resource.",
+            "type": "string"
+          },
+          "name": {
+            "readOnly": true,
+            "description": "Name of the resource.",
+            "type": "string"
+          },
+          "type": {
+            "readOnly": true,
+            "description": "Type of Resource.",
+            "type": "string"
+          },
+          "location": {
+            "description": "Location of the resource.",
+            "type": "string",
+            "enum": [
+              "global"
+            ],
+            "x-ms-enum": {
+              "name": "Location",
+              "modelAsString": true
+            },
+            "x-ms-mutability": [
+              "create",
+              "read"
+            ]
+          },
+          "tags": {
+            "description": "Custom tags for the resource.",
+            "type": "object",
+            "additionalProperties": {
+              "type": "string"
+            }
+          },
+          "etag": {
+            "description": "The entity tag used for optimistic concurrency when modifying the resource.",
+            "type": "string"
+          }
+        },
+        "required": [
+          "location"
+        ],
+        "x-ms-azure-resource": true
+      },
+      "ErrorDetails": {
+        "description": "The details of the error.",
+        "properties": {
+          "code": {
+            "description": "Error code.",
+            "type": "string",
+            "readOnly": true
+          },
+          "message": {
+            "description": "Error message indicating why the operation failed.",
+            "type": "string",
+            "readOnly": true
+          },
+          "target": {
+            "description": "The target of the particular error.",
+            "type": "string",
+            "readOnly": true
+          }
+        }
+      },
+      "ErrorResponse": {
```

Comment:
Please use the default ARM error pattern, as done here: https://github.com/Azure/azure-rest-api-specs/blob/d77baea351ade4453199f73e9aad1f4b8a320a76/specification/resources/resource-manager/Microsoft.Resources/stable/2019-10-01/resources.json#L4418

Reply:
Updated.